samples/drawing/drawing.cpp

```diff
@@ -2790,13 +2790,16 @@ void MyFrame::OnSave(wxCommandEvent& WXUNUSED(event))
             wxGraphicsRenderer* tempRenderer = m_canvas->GetRenderer();
             m_canvas->UseGraphicRenderer(nullptr);
 #endif
-            wxSVGFileDC svgdc(dlg.GetPath(),
-                              canvasSize.GetWidth(),
-                              canvasSize.GetHeight(),
-                              72,
-                              "Drawing sample");
-            svgdc.SetBitmapHandler(new wxSVGBitmapEmbedHandler());
-            m_canvas->Draw(svgdc);
+            wxSize svgSize;
+            wxSVGFileDC tempSvgDC(svgSize);
+            m_canvas->Draw(tempSvgDC);
+
+            svgSize = wxSize(tempSvgDC.MaxX(), tempSvgDC.MaxY());
+            svgSize.IncBy(15); // account for wxPen width exceeding bounds
```

Comment:
This "15" seems pretty arbitrary, but I guess we can't do any better?

Reply:
It is a bit arbitrary, but it seems to work for the drawing and svg samples, while `10` still had some lines cut-off.